documentation/book/proc-accessing-kafka-using-nodeports.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-accessing-kafka-using-nodeports-{context}']
+= Accessing Kafka using node ports routes
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Deploy Kafka cluster with external listener enabled and configured to the type `nodeport`.
++
+An example configuration with an external listener configured to use node ports:
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+spec:
+  kafka:
+    # ...
+    listeners:
+      external:
+        type: nodeport
+        # ...
+    # ...
+  zookeeper:
+    # ...
+----
+
+. Create or update the resource.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl apply -f _<your-file>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc apply -f _<your-file>_
+
+. Find the port number of the bootstrap service.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl get service _<cluster-name>_-kafka-external-bootstrap -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}'
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc get service _<cluster-name>_-kafka-external-bootstrap -o=jsonpath='{.spec.ports[0].nodePort}{"\n"}'
++
+The port should be used in the Kafka _bootstrap_ address.
+
+. Find the address of the {ProductPlatformName} node.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl get node _<node-name>_ -o=jsonpath='{range .status.addresses[*]}{.type}{"\t"}{.address}{"\n"}'
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc get node _<node-name>_ -o=jsonpath='{range .status.addresses[*]}{.type}{"\t"}{.address}{"\n"}'
++
+The command might print different types of addresses.
+Select the right address type according to following order:
++
+.. ExternalDNS
+.. ExternalIP
+.. Hostname
+.. InternalDNS
+.. InternalIP
++
+The address should be used together with the port from previous step in the Kafka _bootstrap_ address.
```

Comment:
Use the address with the same port you used for the Kafka _bootstrap_ address, in step 3.

Reply:
@laidan6000 I'm not sure whether this carries the same meaning. Basically we find the port in step 3. and the hostname in this step. And together they form the bootstrap address. So _with the same port you used for the Kafka bootstrap address, in step 3_ IMHO doesn't make sense. WDYT?

Reply:
That makes sense @scholzj .

How about:

Use the address with the port in the previous step found in the Kafka _bootstrap_ address.